content/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added.md

```diff
@@ -30,38 +30,38 @@ In the tutorial, you will first make a workflow file that uses the [`peter-evans
 1. Copy the following YAML contents into your workflow file.
 
     ```yaml copy
-    {% data reusables.actions.actions-not-certified-by-github-comment %}
-
-    {% data reusables.actions.actions-use-sha-pinning-comment %}
-
     name: Add comment
     on:
       issues:
         types:
           - labeled
     jobs:
       add-comment:
-        if: github.event.label.name == 'help-wanted'
+        if: github.event.label.name == 'help wanted'
         runs-on: ubuntu-latest
         permissions:
           issues: write
         steps:
           - name: Add comment
-            uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
-            with:
-              issue-number: {% raw %}${{ github.event.issue.number }}{% endraw %}
-              body: |
-                This issue is available for anyone to work on. **Make sure to reference this issue in your pull request.** :sparkles: Thank you for your contribution! :sparkles:
+            # https://cli.github.com/manual/gh_issue_comment
+            run: gh issue comment -R "$REPOSITORY" "$NUMBER" --body "$BODY"
+            env:
+              GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
              GH_TOKEN: ${{ github.token }}
```

I know that works. Because [I tested it](https://github.com/peterbe/github-action-tricks/issues/53#issuecomment-1781171488) :)
But what I'm not sure of is if it's established best practice in documentation.

Reply:
is there precedence in the docs one way or the other? 🤔 for users, secrets.GITHUB_TOKEN is 8x more popular than github.token from preliminary searching https://github.com/search?q=secrets.GITHUB_TOKEN&type=code https://github.com/search?q=github.token&type=code 🤷‍♀️

Reply:
I honestly don't know. I vaguely remember that `github.token` is new(er). 

I think it's for the @github/docs-content reviewer to try to find that out. It would certainly be good to have a one blessed choice given that they appear to accomplish the same thing. 

Personally, I find `github.token` preferred because it's clearly something I don't mess with. Whereas if I, as a user, create a secret called `SECRET_INGREDIENT` it would be `${{ secrets.SECRET_INGREDIENT }}` that I use. So it feels like the `secrets. ...` is the nomenclature for *your* custom secrets. 

It feels like it says: "You can create and manage your own secrets. But I've created one for you that you can't create and manage."

Reply:
We typically recommend `secrets.GITHUB_TOKEN`